app/signup/page.tsx

```
@@ -26,16 +34,55 @@ export default function SignupPage() {
+		if (file.size > 5 * 1024 * 1024) {
+			quality = 0.4;
+		} else if (file.size < 2 * 1024 * 1024) {
+			quality = 0.6;
+		} else {
+			quality = 0.8;
+		}
+
+		new Compressor(file, {
+			quality,
+			success: (compressedFile) => {
+				const reader = new FileReader();
```

Biome (1.9.4)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)